presto-main/src/main/java/io/prestosql/sql/rewrite/ShowQueriesRewrite.java

```diff
@@ -547,14 +547,17 @@ protected Node visitShowCreate(ShowCreate node, Void context)
             for (Map.Entry<String, Object> propertyEntry : properties.entrySet()) {
                 String propertyName = propertyEntry.getKey();
                 Object value = propertyEntry.getValue();
-                if (value == null) {
-                    throw new PrestoException(errorCode, format("Property %s for %s cannot have a null value", propertyName, toQualifiedName(objectName, columnName)));
-                }
 
                 PropertyMetadata<?> property = allProperties.get(propertyName);
                 if (property == null) {
-                    throw new PrestoException(errorCode, "No PropertyMetadata for property: " + propertyName);
+                    // ignoring unknown property in SHOW CREATE TABLE
```

Comment:
As an alternative we could only skip properties which are explicitly marked as hidden. This raises the question how to mark those as hidden. If want to go that way, I suggest to encode "hidden" bit in the property name to not blot code too much (e.g. threat properties starting with `_` as hidden). WDYT?